docs/en/workshop/a-molecular-unfolding/evaluate-your-own-model.md

```diff
@@ -66,11 +66,10 @@ If you want to fully customize the evaluation code, follow below steps to make c
 
 1. Make changes to source code
 
-1. Update `quicksight_user` and `default_code_repository` in file `source/cdk.context.json`
+1. Add `default_code_repository` in file `source/cdk.context.json`
```

Comment:
we should remove file `source/cdk.context.json`

Reply:
removed